src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -38,16 +39,32 @@ class FlexMicroform extends React.Component {
       });
       return;
     }
-    window.microform = new window.Flex(captureKeyId).microform({
-      styles: {
-        input: {
-          'font-size': '16px',
-          'font-family': 'Roboto, "Helvetica Neue", Arial, sans-serif',
-          color: '#2d323e',
+    try {
+      window.microform = new window.Flex(captureKeyId).microform({
+        styles: {
+          input: {
+            'font-size': '16px',
+            'font-family': 'Roboto, "Helvetica Neue", Arial, sans-serif',
+            color: '#2d323e',
+          },
         },
-      },
-    });
-    this.props.dispatch(microformStatus(STATUS_READY));
+      });
+      this.props.dispatch(microformStatus(STATUS_READY));
+    } catch (err) {
+      if (err.reason === 'CAPTURE_CONTEXT_EXPIRED' && this.state.captureContextRetryCount < 5) {
+        this.captureContextRetryCount++;
```

Comment:
Doesn't this need a `setState`?

Reply:
I think this shouldn't be in state. Should just be an instance variable on the object.